broker/src/main/java/org/apache/rocketmq/broker/processor/AdminBrokerProcessor.java

```diff
@@ -412,10 +412,12 @@ private synchronized RemotingCommand updateAndCreateTopic(ChannelHandlerContext
             response.setRemark(result.getRemark());
             return response;
         }
-        if (TopicValidator.isSystemTopic(topic)) {
-            response.setCode(ResponseCode.SYSTEM_ERROR);
-            response.setRemark("The topic[" + topic + "] is conflict with system topic.");
-            return response;
+        if (brokerController.getBrokerConfig().isValidateSystemTopicWhenUpdateTopic()) {
+            if (TopicValidator.isSystemTopic(topic)) {
```

Comment:
Some other places, such as updateAndCreateStaticTopic, also use TopicValidator#isSystemTopic. Should we deal with them in the same way?

Reply:
It's reasonable.